From 9bc1e488515722c844bec286e25ca99aa7694404 Mon Sep 17 00:00:00 2001
From: Andrei Cherechesu <andrei.cherechesu@nxp.com>
Date: Thu, 3 Feb 2022 18:58:04 +0200
Subject: [PATCH 48/50] bl2: Change compatible of U-Boot DT '/ddr' node

Changed the compatible of the U-Boot's DT '/ddr' node to
align with U-Boot and Linux.

Issue: ALB-8353
Upstream-Status: Pending 

Signed-off-by: Andrei Cherechesu <andrei.cherechesu@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 plat/nxp/s32/s32_bl2_el3.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/plat/nxp/s32/s32_bl2_el3.c b/plat/nxp/s32/s32_bl2_el3.c
index 59ac40459..b0673a0d5 100644
--- a/plat/nxp/s32/s32_bl2_el3.c
+++ b/plat/nxp/s32/s32_bl2_el3.c
@@ -345,15 +345,15 @@ static int ft_fixup_ddr_errata(void *blob)
 		return 0;
 	}
 
-	nodeoff = fdt_node_offset_by_compatible(blob, -1, "fsl,s32gen1-ddr-err050543");
+	nodeoff = fdt_node_offset_by_compatible(blob, -1, "fsl,s32gen1-ddr");
 	if (nodeoff < 0) {
-		ERROR("Failed to get offset of 'fsl,s32gen1-ddr-err050543' node\n");
+		ERROR("Failed to get offset of 'fsl,s32gen1-ddr' node\n");
 		return nodeoff;
 	}
 
 	ret = fdt_setprop_string(blob, nodeoff, "status", "okay");
 	if (ret) {
-		ERROR("Failed to enable 'fsl,s32gen1-ddr-err050543' node\n");
+		ERROR("Failed to enable 'fsl,s32gen1-ddr' node\n");
 		return ret;
 	}
 
-- 
2.17.1

